### PR TITLE
Update inifile.rst for NAT default value

### PR DESCRIPTION
**Pull request by farmio**
Since I was not expecting #567 to be merged so quickly, I didn't include an update to the documentation in the first commit 😬.
Let me know if there are more files to be updated accordingly since I'm not that familiar with the codebase.

### CODE REVIEW

File: doc/inifile.rst [hunk 1]
```diff
@@ -490,15 +490,16 @@ The following options are not recognized unless "nat" is set.
 
 * nat-ip (string: IP address)
 
-  ??
+  Target IP address for the tunnelling servers responses and requests.
+  When set to zero the server shall use the IP address in the IP package received.
```
Comment (dallmair) [inline doc/inifile.rst:494]: Proposal: Replace "package" with "packet".

```suggestion
  When set to zero the server shall use the IP address in the IP packet received.
```

File: doc/inifile.rst [hunk 2]
```diff
@@ -490,15 +490,16 @@ The following options are not recognized unless "nat" is set.
...
   
-  Mandatory if "nat" is set, otherwise disallowed.
+  Defaults to "0.0.0.0" if "nat" is set, otherwise disallowed.
```
Comment (dallmair) [inline doc/inifile.rst:496]:
```suggestion
  Defaults to "0.0.0.0" if "nat" is set, otherwise ignored.
```

File: doc/inifile.rst [hunk 3]
```diff
@@ -490,15 +490,16 @@ The following options are not recognized unless "nat" is set.
...
 
 * data-port (int)
 
-  ??
+  Target port number for the tunnelling servers responses and requests.
   
-  Mandatory if "nat" is set, otherwise disallowed.
+  Defaults to 0 if "nat" is set, otherwise disallowed.
```
Comment (dallmair) [inline doc/inifile.rst:502]:
```suggestion
  Defaults to 0 if "nat" is set, otherwise ignored.
```

**Comment by dallmair** [comment]
@smurfix In general this LGTM, modulo tiny wording suggestions. Please merge.

File: doc/inifile.rst (hunk 1 above)
Comment (farmio) [reply doc/inifile.rst]: Fine for me. I was copying this from KNX specs.

**Pull request merged**